### PR TITLE
docs(CodeSandbox): fix styles

### LINKED ISSUE #1848: Some examples rendered unexpectedly on CodeSandbox
### Steps
1. Navigate to https://stardust-react-6rtbg2t0t.now.sh/components/form/definition#types-form
2. Go to CodeSandbox
3. Observe example not rendered in the same way as in docs

### Expected Result
image

### Actual Result
image

### Version
master 84be797d22c9a0576bbaf7c2c84507a16e4e71db

### PR DESCRIPTION
**Pull request by lucivpav**
Fixes #1848 

### Before
image

### After
image

### CODE REVIEW

File: packages/code-sandbox/src/SandboxApp.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const SandboxApp: React.FunctionComponent = props => {
   const [theme, setTheme] = React.useState(items[0].value)
 
   return (
-    <Provider theme={themes[theme]} styles={{ height: '100vh', padding: '1rem' }}>
+    <Provider theme={themes[theme]} styles={{ padding: '0 0 0 1rem' }}>
```
Comment (layershifter) [inline packages/code-sandbox/src/SandboxApp.tsx]: Have you tried this with other themes?

Comment (lucivpav) [reply packages/code-sandbox/src/SandboxApp.tsx]: image

Comment (lucivpav) [reply packages/code-sandbox/src/SandboxApp.tsx]: It works now
image

File: packages/code-sandbox/src/SandboxApp.tsx [hunk 2]
```diff
@@ -27,30 +27,32 @@ const SandboxApp: React.FunctionComponent = props => {
...
-        <Text as="a" href="https://stardust-ui.github.io/react/">
-          our docs
-        </Text>{' '}
-        and{' '}
-        <Text as="a" href="https://github.com/stardust-ui/react">
-          GitHub
-        </Text>
-        .
-      </p>
+      <Provider theme={themes[theme]}>
```
Comment (layershifter) [inline packages/code-sandbox/src/SandboxApp.tsx]:
```suggestion
      <div>
```

Plain `div` does the same effect:
https://codesandbox.io/s/stardust-ui-example-c2lqb

**Review by layershifter — APPROVED** [review]

**Pull request merged**